cmd/syft/cli/commands.go

```diff
@@ -75,6 +77,11 @@ func New() (*cobra.Command, error) {
 	if err != nil {
 		return nil, err
 	}
+	// convert also uses flags from the packagesCmd since it generates an sbom
+	err = po.AddFlags(convertCmd, v)
```

Comment:
There's not a lot in the PR description so apologies if I missed this in the linked issue, but why should `covert` be able to generate an SBOM? 

```
syft convert sbom.syft.json -o cyclonedx-json=sbom.cdx.json
```

I don't see any of the common packages flags here and lean towards keeping convert pure as it's own command with it's own flags atm.

Reply:
You're right, we can keep it isolated flag-wise.